instrumentation/geode-1.4/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/geode/PutGetTest.java

```diff
@@ -0,0 +1,203 @@
+  }
+
+  @ParameterizedTest
+  @MethodSource("provideParameters")
+  void testExistsValue(Object key, Object value) throws QueryException {
+    boolean cacheValue =
+        testing.runWithSpan(
+            "someTrace",
+            () -> {
+              region.clear();
+              region.put(key, value);
+              return region.existsValue("SELECT * FROM /test-region");
+            });
+    assertTrue(cacheValue);
+    assertGeodeTrace("existsValue", "SELECT * FROM /test-region");
+  }
+
+  @Test
+  void shouldSanitizeGeodeQuery() throws QueryException {
+    Card value = new Card("1234432156788765", "10/2020");
+    SelectResults<Object> results =
+        testing.runWithSpan(
+            "someTrace",
+            () -> {
+              region.clear();
+              region.put(1, value);
+              return region.query("SELECT * FROM /test-region p WHERE p.expDate = '10/2020'");
+            });
+
+    assertEquals(value, results.asList().get(0));
+    assertGeodeTrace("query", "SELECT * FROM /test-region p WHERE p.expDate = ?");
+  }
+
+  void assertGeodeTrace(String verb, String query) {
+    testing.waitAndAssertTraces(
+        trace ->
+            trace
+                .hasSize(4)
+                .hasSpansSatisfyingExactly(
+                    span -> span.hasName("someTrace").hasKind(SpanKind.INTERNAL),
+                    span ->
+                        span.hasName("clear test-region")
+                            .hasKind(SpanKind.CLIENT)
+                            .hasAttributesSatisfyingExactly(
+                                equalTo(SemanticAttributes.DB_SYSTEM, "geode"),
+                                equalTo(SemanticAttributes.DB_NAME, "test-region"),
+                                equalTo(SemanticAttributes.DB_OPERATION, "clear")),
+                    span ->
+                        span.hasName("put test-region")
+                            .hasKind(SpanKind.CLIENT)
+                            .hasAttributesSatisfyingExactly(
+                                equalTo(SemanticAttributes.DB_SYSTEM, "geode"),
+                                equalTo(SemanticAttributes.DB_NAME, "test-region"),
+                                equalTo(SemanticAttributes.DB_OPERATION, "put")),
+                    span ->
+                        span.hasName(verb.concat(" test-region"))
+                            .hasKind(SpanKind.CLIENT)
+                            .hasAttributesSatisfying(
+                                attributes -> {
+                                  assertThat(attributes)
+                                      .containsEntry(SemanticAttributes.DB_SYSTEM, "geode");
+                                  assertThat(attributes)
+                                      .containsEntry(SemanticAttributes.DB_NAME, "test-region");
+                                  assertThat(attributes)
+                                      .containsEntry(SemanticAttributes.DB_OPERATION, verb);
```

Comment:
Nit: you can use the fluent interface and avoid calling `assertThat(attributes)` multiple times
```suggestion
                                  assertThat(attributes)
                                      .containsEntry(SemanticAttributes.DB_SYSTEM, "geode")
                                      .containsEntry(SemanticAttributes.DB_NAME, "test-region")
                                      .containsEntry(SemanticAttributes.DB_OPERATION, verb);
```